docs/docs/20-usage/20-workflow-syntax.md

```diff
@@ -594,20 +594,34 @@ For more details check the [matrix build docs](./30-matrix-workflows.md).
 
 ## `labels`
 
-You can set labels for your workflow to select an agent to execute the workflow on. An agent will pick up and run a workflow when **every** label assigned to it matches the agents labels.
-
-To set additional agent labels, check the [agent configuration options](../30-administration/10-configuration/30-agent.md#agent_labels). Agents will have at least four default labels: `platform=agent-os/agent-arch`, `hostname=my-agent`, `backend=docker` (type of the agent backend) and `repo=*`. Agents can use a `*` as a wildcard for a label. For example `repo=*` will match every repo.
-
-Workflow labels with an empty value will be ignored.
-By default, each workflow has at least the `repo=your-user/your-repo-name` label. If you have set the [platform attribute](#platform) for your workflow it will have a label like `platform=your-os/your-arch` as well.
-
-You can add additional labels as a key value map:
+Use labels to select the agent that executes your workflow. An agent will execute a workflow only if **all** its assigned labels match the workflow's labels. For Kubernetes agents, these labels propagate to any related resources created during pipeline execution, including pods and secrets.
+To configure additional agent labels, see the [agent configuration options](../30-administration/15-agent-config.md#woodpecker_agent_labels). Agents have pre-configured filters for the following labels:
```

Comment:
This line causes the docs build error:

```
        │ Cause: Docs markdown link couldn't be resolved: (../30-administration/15-agent-config.md) in source file "/woodpecker/src/github.com/woodpecker-ci/woodpecker/docs/docs/20-usage/20-workflow-syntax.md" for version current
```

We have refactored the admin docs, the new path is:

```suggestion
To configure additional agent labels, see the [agent configuration options](../30-administration/10-configuration/30-agent.md#agent_labels). Agents have pre-configured filters for the following labels:
```